scripts/benchmarks/preflight_kha348.py

```
@@ -0,0 +1,95 @@
+        if sha in log:
+            print(f"[preflight] KHA-348 fix commit found in git log: {sha}", flush=True)
+            return True
+    print("[preflight] ERROR: KHA-348 fix commits not found in last 200 log entries.", file=sys.stderr)
+    print(f"[preflight] Expected one of: {KHA348_FIX_COMMITS}", file=sys.stderr)
+    print("[preflight] Run: git fetch origin main && git merge origin/main", file=sys.stderr)
+    return False
+
+
+def check_main_branch() -> bool:
+    """Warn (non-blocking) if we're not on main."""
+    rc, branch = run(["git", "rev-parse", "--abbrev-ref", "HEAD"])
```

Ruff (0.15.12)

[warning] Unpacked variable `rc` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)